specification/components/schemas/eRS-Appointment.yaml

```diff
@@ -107,6 +107,34 @@ properties:
                     type: string
                     nullable: true
                     example: 'Provider Converted Advice and Guidance, Admin to Refer'
+              - type: object
+                title: eRS-Cancellation-Reason
+                required:
+                  - system
+                  - code
+                properties:
+                  system:
+                    type: string
+                    enum:
+                      - _baseurl_/STU3/CodeSystem/APPOINTMENT-CANCELLATION-REASON
+                    example: '_baseurl_/STU3/CodeSystem/APPOINTMENT-CANCELLATION-REASON'
+                  code:
+                    type: string
+                    enum:
+                      - INTEND_PRIVATE
```

Comment:
As this is a dynamic code system I don't think we should be defining the enum values here as they could change in the future. Instead I'd probably just include an example.

Reply:
Jack, We had discussed the similar thing for Healthcare Service endpoint that return Appointment Type Refdata and we have created the enum in java side. Though, it is dynamic enum from refdata,  we decided to create the consent enum and agreed to do future changes if new refdata value is added. Because of the discussion, I put the consent enum here. Do you still think we should use dynamic?